## Cost and Benefit Analysis

### Cost Analysis

Zenivex is an approved pesticide for adult mosquito control by the U.S. Environmental Protection Agency ([*source*](https://www.chicago.gov/city/en/depts/cdph/provdrs/healthy_communities/news/2023/august/coc-spraying-for-mosquitoes-aug-16.html)). It is widely employed by the Chicago Department of Public Health in outdoor residential and recreational spaces to effectively manage mosquito populations.


Spray rate of Zenivex will be at a rate of 1.5 ounces per acre([*source*](https://www.nbcchicago.com/news/local/city-to-spray-mosquito-killing-insecticide-wednesday-night/64951/))

Size of Chicago (land) is 589.82 km ([*source*](https://en.wikipedia.org/wiki/Chicago)).
Converting into acres will be: 589.82 square km =145,747.67 acres

Cost for Zenivex per gallon: 11,595.95 / 30 = 386.53 USD

Spray Amount: 145,747.67 / 1.5 = 97,165.11 ounces
Spray Amount in Gallons: 97,165.11 /128= 759.10 gallon

Cost of Zenivex required: 759.10 x 386.53 USD =  293,415.87 USD

The frequency for spraying is not fixed. But according to the articles for spraying activites for 2023 ([*source*](https://www.chicago.gov/city/en/depts/cdph/provdrs/healthy_communities/news/2023/august/coc-spraying-for-mosquitoes-aug-16.html)) and 2021 ([*source*](https://www.chicago.gov/city/en/depts/cdph/provdrs/healthy_living/news/2021/august/city-to-spray-insecticide-wednesday-to-kill-mosquitoes.html)) , there has only been 1 spray activity reported for the respective years.

Hence, the frequency for spraying will be assumed to be once a year.

Total cost for Zenivex: **293,415.87 USD**

### Consumer Price Index (CPI)

CPI will be factored into the cost for the Benefit Analysis in the later portion of this report.
The data comprises of the monthly CPI figures from year 2012 to year 2023 ([*source*](https://www.bls.gov/cpi/factsheets/medical-care.htm)).

In [1]:
import numpy as np
import pandas as pd

In [2]:
cpi = pd.read_excel('cpi.xlsx')

C:\Users\Lua Yeow Wen\anaconda3\Lib\site-packages\openpyxl\styles\stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")


In [3]:
# Remove the first 11 rows
cpi = cpi.drop(range(10))


# Reset the index after dropping rows
cpi = cpi.reset_index(drop=True)


# Set First Row as Headers
cpi.columns = cpi.iloc[0]

# Drop the first row after setting it as headers
cpi = cpi.drop(0)

# Reset the index after dropping rows
cpi = cpi.reset_index(drop=True)

cpi = cpi.drop(['HALF1', 'HALF2'], axis=1)


#Convert from Wide to Long data
cpi = pd.melt(cpi, id_vars=['Year'], var_name='Month', value_name='Value')

# Get Average Annual CPI
cpi = cpi.groupby('Year')['Value'].mean().reset_index()

In [4]:
cpi

Year       Value
0   2012  414.923917
1   2013  425.134500
2   2014  435.292250
3   2015  446.752083
4   2016  463.674667
5   2017  475.321833
6   2018  484.707000
7   2019  498.413333
8   2020  518.875500
9   2021  525.276417
10  2022  546.553500
11  2023  548.221778

### Benefit Analysis

Consumer Price Index (CPI) will be factored into median household income and treatment costs for WNV.

The median household income in Chicago is 68,654.48 USD (65,781/ 525.28 x 548.22), assuming that the income aligns with the CPI ([*source*](https://www.census.gov/quickfacts/fact/table/chicagocityillinois/INC110221#INC110221)).


**Number of reported cases in:**

* 2023: 20 reported WNV cases ([*source*](https://www.chicago.gov/content/dam/city/depts/cdph/Mosquito-Borne-Diseases/CDPH-Arboviral-Report-2023-WK39.pdf))
* 2022: 7 reported WNV cases ([*source*](https://www.chicagohan.org/alert-detail/-/alert-details/46681285?p_r_p_categoryId=undefined))
* 2021: 13 reported WNV cases ([*source*](https://www.chicago.gov/city/en/depts/cdph/provdrs/healthy_communities/news/2022/september/first-human-case-of-west-nile-virus-detected-in-chicago-in-2022.html#:~:text=CHICAGO%20%E2%80%93%20The%20Chicago%20Department%20of,human%20cases%2C%20and%20no%20deaths.))


Although there was not many WNV cases reported in the past 3 years, but if the officials are complacent, the city is likely to face the **high number of WNV cases as high as 229** in the year 2012 ([*source*](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7241786/)). Hence, the number of reported cases in 2012 will be used for this analysis.


**Breakdown of reported cases:**

80% of individuals with symptoms encounter an acute systemic febrile illness ([*source*](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3945683/)), referred to as West Nile fever or non-neuroinvasive disease, which commonly includes symptoms like headache, myalgia, arthralgia, or rash.

Neuroinvasive disease, a more severe manifestation of the infection, develops in less than 1% of infected individuals and typically presents as either meningitis or encephalitis.


* 229 x 80% = approx 184 asymptomatic individuals
* 229 x 1% = aprrox 2 individuals experience neuroinvasive disease, commonly presenting as either meningitis or encephalitis
* 229 - 184 - 2 = approx 43 individuals undergo an acute systemic febrile illness referred to as West Nile fever or non-neuroinvasive disease, which frequently involves symptoms such as headache, myalgia, arthralgia, or rash.


**There are treatment two costs for individuals to bear ([*source*](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3945683/)):**
* Initial cost such as initial hospital and lost productivity costs.
* Long-term costs such as medical appointments, additional care costs, medicines, equipment, or modifications, lost productivity.

**The costs were factored with CPI:**
* Initial costs for non-severe case: approx 6,100.25 USD (4,617/414.92 x 548.22)
* Initial costs for severe case: approx 33,186.05 USD (25,117/414.92 x 548.22)

* Long-term costs for non-severe case: approx 3,000.58 USD (2,271/414.92 x 548.22)
* Long-term costs for severe case: approx 29,897.44 USD (22,628/414.92 x 548.22)



**Total treatment costs for non-severe case per person:**

* 6,100.25 + 3,000.58 = 9,100.83 USD

**Total treatment cost for non-severe case in Chicago:**

* 43 x 9,100.83 = 391,335.69 USD


**Total treatment costs for severe case per person:**

* 33,186.05 + 29,897.44  = 63,083.49 USD


**Total treatment cost for severe case in Chicago:**

* 2 x 63,083.49 = 126,166.98 USD

Total treatment cost: approx **517,502.67 USD**

**Loss of Income per person (Assuming an individual is unable to work for 14 days):**
* 14 x (68,654.48 median household income /260 working days) = approx 3,696.78 USD

**Loss of Income in Chicago:**
* 3,696.78 x (43 + 2) = approx **166,355.09 USD**


**Total Loss:**
* Total treatment cost + Loss of Income in Chicago:

517,502.67 + 166,355.09 = approx **683,857.76 USD**


### Conclusion
    
Based on historical data (year 2007, 2009, 2011 and 2013) , the presence of WNV virus is based on these factors:

* Numbers of mosquitoes.
* Temperature and humidity is higher than normal days.
* Lower average wind speed.
* During the months from May to August.
* Negligence of spraying at some of the areas with WNV presence.
* Mosquitoes species belonging to Culex Pipiens and Culex Restuans.


The top locations with highest WNV presence of at least 14 instances are:

* ORD Terminal 5, O'Hare International Airport, Chicago, IL 60666, USA
* South Doty Avenue, Chicago, IL, USA 
* 4100 North Oak Park Avenue, Chicago, IL 60634, USA
* South Stony Island Avenue, Chicago, IL, USA
* 4600 Milwaukee Avenue, Chicago, IL 60630, USA

The top traps located that has high mosquitoes count and WNV presence are T900 and T115.

Thunderstorms and mist might contribute to no. of WNV cases.

Limitations:
* The sprayed data is not consistent.
* Unavailability of manpower cost and data for cost analysis.

Based on the reports/articles found online, the total material cost for Zenivex is **293,415.87 USD**.

The total cost for treating WNV in Chicago is estimated at **517,502.67 USD**. Also, people who have been infected with West Nile Virus (WNV) may experience a reduction in their income, resulting in a cumulative income loss of **166,355.09 USD** in the city.
The total estimated loss in Chicago is **683,857.76 USD**.


However, there are other factors that may further contribute to the city's economy negatively such as lower workplace productivity, reduced tourism and increase of moving to other cities due to the presence of WNV.

Even though there are other factors such as manpower cost is not included in the cost analysis due to the unavailbilty of data, the officials should not underestimate the long term negative effect for not spraying insecticide. Hence, the efforts for spraying the city should not be stopped.


### Reccomendation
* To use data driven approach to reduce WNV by implementing preventive measures (refer to the first two points below).

* The best period for spraying insecticide is during the month of July to August as these two months are the peak of the mosquitoes numbers.

* Ensure thorough spraying, as there are regions with the presence of West Nile Virus that have not been treated with spray and we do not assume that the pressence of WNV is not caused by mosquitoes.

* The officials should also be cautious that the wrong concentration amount of insecticides may cause the mosquitoes to grow resistance ([*source*](https://www.vox.com/health/23814358/west-nile-virus-symptoms-mosquito-repellant-disease-bite-insecticide-resistance)).

* Residents to wear loose-fitting clothes that can cover arms and legs during the month of July and August([*source*](https://www.cdc.gov/westnile/prevention/index.html)).

* Educate the residents to prevent mosquitoes breeding, such as draining water out from anything that can collect rainwater.

* Enforce the laws and regulations for mosquitoes breeding.

### Further Improvements
 * More data on number of mosquitoes for more accurate modelling results.


